common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalBlockStoreClient.java

```diff
@@ -103,14 +103,22 @@ public void fetchBlocks(
     try {
       RetryingBlockFetcher.BlockFetchStarter blockFetchStarter =
           (blockIds1, listener1) -> {
-            // Unless this client is closed.
-            if (clientFactory != null) {
-              TransportClient client = clientFactory.createClient(host, port);
```

Comment:
shall we only add try-catch to wrap this line?

Reply:
> shall we only add try-catch to wrap this line?

It's OK to only wrap this line.

Reply:
> shall we only add try-catch to wrap this line?

Updated, hope for review current change